examples/pwa/pwa-ampdoc-shell.html

```diff
@@ -0,0 +1,141 @@
+<!doctype html>
+<html>
+<head>
+  <meta charset="utf-8">
+  <title>PWA</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1,minimum-scale=1,maximum-scale=1,user-scalable=no,minimal-ui">
+  <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Georgia|Open+Sans|Roboto">
+
+  <style amp-boilerplate>body{-webkit-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-moz-animation:-amp-start 8s steps(1,end) 0s 1 normal both;-ms-animation:-amp-start 8s steps(1,end) 0s 1 normal both;animation:-amp-start 8s steps(1,end) 0s 1 normal both}@-webkit-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-moz-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-ms-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@-o-keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}@keyframes -amp-start{from{visibility:hidden}to{visibility:visible}}</style><noscript><style amp-boilerplate>body{-webkit-animation:none;-moz-animation:none;-ms-animation:none;animation:none}</style></noscript>
+  <script async custom-element="amp-sidebar" src="https://cdn.ampproject.org/v0/amp-sidebar-0.1.js"></script>
+
+  <!-- pwa-ampdoc-shell.html should run at `/pwa-ampdoc-shell` -->
+  <script>
+    if (window.location.pathname.indexOf('/examples') == 0) {
+      window.location.assign('/pwa/ampdoc-shell');
+    } else {
+      window.AMP_CONFIG = {
```

Comment:
This is ok to start with, but for PROD uses we'll need to switch to, e.g., meta element for the experiment:

<meta name="amp-experiments-opt-in" content="ampdoc-shell">
Otherwise, it'll be super racy and will override other useful config values. To do this, you will need to whitelist ampdoc-shell in the allow-doc-opt-in of canary-config.json and prod-config.json. Would you mind sending a catch up PR with these so that we don't forget this later?

Reply:
Ok, will do. I tried to add it in this PR but I didn't know it had to be done in a separate PR